content/class/Post.md

```diff
@@ -0,0 +1,316 @@
+
+```c#
+public string id { get; }
+```
+
+The unique identifier a Post has on the platform.
+
+| API   | Twitter 1 | Twitter 2 | Mastodon |
+| ----- | --------- | --------- | -------- |
+| Field | `id_str`  | `id`      | `id`     |
+| Type  | string    | string    | string   |
+
+### `type` {#property_type}
+
+```c#
+public PostType type { get; }
+```
+
+Defines what type this Post is. See [`PostType`](../enum/PostType.md).
+
+### `text` {#property_text}
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2](https://img.shields.io/badge/API-Twitter%202-blue?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+```c#
+public string text { get; }
+```
+
+The text of the Post formatted as am markup-text for the usage in the UI.
+
+> This is the final version formatted for the use in the UI, formatted by [`format_text`](../../utils/namespace/ParseUtils.md#method_format_text), while the original text is found in [`raw_text`](#field_raw_text).
+
+### `author` {#property_author}
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2](https://img.shields.io/badge/API-Twitter%202-blue?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+```c#
+public User author { get; }
+```
+
+The [`User`](User.md) which posted this post.
+
+| API   | Twitter 1 | Twitter 2   | Mastodon  |
+| ----- | --------- | ----------- | --------- |
+| Field | `user`    | `author_id` | `account` |
+| Type  | object    | id string   | object    |
+
+### `date` {#property_date}
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2](https://img.shields.io/badge/API-Twitter%202-blue?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+```c#
+public GLib.DateTime date { get; }
+```
+
+The date this Post was posted on the platform.
+
+| API   | Twitter 1    | Twitter 2       | Mastodon        |
+| ----- | ------------ | --------------- | --------------- |
+| Field | `created_at` | `created_at`    | `created_at`    |
+| Type  | string       | ISO 8601 string | ISO 8601 string |
+
+### `url` {#property_url}
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2](https://img.shields.io/badge/API-Twitter%202-blue?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+```c#
+public string url { get; }
+```
+
+A url to open the Post on the website of the used provider.
+
+> Twitter urls have to be generated internally, as there is no field for it.
+>
+> So, not really neccessary for Twitter (as it is only `twitter.com`, which can be generated on the fly), but as Mastodon has multiple servers, we need to specify it for it, should it be implemented.
```

Comment:
Not necessary to have an internal backing store holding the string (unless we're going to call it a lot and benefit from caching it, which seems unlikely). But it does make sense to have it as a consistent property.